pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -80,22 +74,16 @@ RUN apt-get update \
     build-essential \
     ca-certificates \
     cmake \
-    cuda-command-line-tools-11-7 \
-    cuda-cudart-11-7 \
-    cuda-libraries-11-7 \
+    cuda-toolkit-11-8 \
+    cuda-command-line-tools-11-8 \
+    cuda-nvcc-11-8 \
+    libcudnn8=$CUDNN_VERSION-1+cuda11.8 \
     curl \
     emacs \
     git \
     hwloc \
     jq \
-    libcublas-11-7=${CUBLAS_VERSION}-1 \
-    libcublas-dev-11-7=${CUBLAS_VERSION}-1 \
-    libcudnn8=$CUDNN_VERSION-1+cuda11.7 \
-    libcufft-dev-11-7 \
-    libcurand-dev-11-7 \
```

Comment:
this is to reduce the docker size right? LGTM, let's also do a `mamba clean -y -a -f` at the end to clean conda caches